src/Curl/Curl.php

```diff
@@ -543,8 +543,10 @@ public function setDigestAuthentication($username, $password = '')
      */
     public function setCookie($key, $value)
     {
-        $this->cookies[$key] = $value;
-        $this->setOpt(CURLOPT_COOKIE, str_replace(' ', '%20', urldecode(http_build_query($this->cookies, '', '; '))));
+        $this->cookies[$key] = $value;        
+        $this->setOpt(CURLOPT_COOKIE, implode('; ', array_map(function($name) { 
+            return $name . '=' . str_replace(' ', '%20', $this->cookies[$name]); 
```

Comment:
> Fatal error: Using $this when not in object context in /home/travis/build/php-curl-class/php-curl-class/src/Curl/Curl.php on line 548

Can we use `use` for the `array_map` function for PHP 5.3 compatibility?

```php
array_map(function($name) use ($instance) { ... }
```

https://travis-ci.org/php-curl-class/php-curl-class/jobs/87628295